packages/web-components/src/components/ic-button/ic-button.tsx

```diff
@@ -80,11 +79,6 @@ export class Button {
    */
   @Prop() disabled?: boolean = false;
 
-  /**
-   * If `true`, the ic-tooltip which is shown for icon variant will be disabled. Title or aria-label must be set if this prop is not applied.
-   */
-  @Prop() disableTooltip?: boolean = false;
```

Comment:
I think instead of completely deleting this prop, we need to deprecate it in case people are currently using it. Then we can delete it when we come to do v3 work.

Reply:
Done :)